lms/templates/seq_module.html

```diff
@@ -23,6 +23,13 @@
     </button>
     <nav class="sequence-list-wrapper" aria-label="${_('Sequence')}">
       <ol id="sequence-list" role="tablist">
+        % if gate_content:
```

Comment:
Nit: consider using consistent terminology and changing this to `gated_content`.

Reply:
@nasthagiri 
changed to gated_content